src/main/java/DAO/DepartmentDAO.java

```diff
@@ -133,7 +133,7 @@ public List<Department> getAllDepartments(){
                 }
 
                 // create a department
-                UUID id = UUID.fromString(departmentResult.getString("ID"));
+                UUID id = UUID.fromString(departmentResult.getString("ID").trim());
```

Comment:
Why trim? Wouldn't this cause issues with the existing entries in the database (projects, employees, etc.)?

Reply:
Without trim there were issues logging in for me.

Reply:
It's a fail-safe.